docs/omero_figure_scripting.rst

```diff
@@ -13,44 +13,43 @@ Setup
 
 -  Install the OMERO.figure web app as described at https://pypi.org/project/omero-figure/
 -  Upload the script :download:`Split_View_Figure.py <../scripts/Split_View_Figure.py>` to the OMERO scripting service
+-  For the `Shapes heatmap from OMERO.table` example, you'll need to draw ROIs on an Image and create an OMERO.table on the
+   Image with a ``Shape`` column with corresponding Shape IDs, as described at `omero-metadata <https://github.com/ome/omero-metadata#populate>`_
```

Comment:
I do not see. any mention of ``Shape`` in https://github.com/ome/omero-metadata#populate - nor is there such in any open PR.  I think we should add it there. But even better would be an addition of an example CSV directly to this workflow. We have a precedent of this in https://omero-guides.readthedocs.io/en/latest/upload/docs/metadata.html, I mean the https://omero-guides.readthedocs.io/en/latest/_downloads/8227ce5769b3477eff58853580d44493/four-images.csv - for that, the CSV does not need to be the one used in idr0079 workflow, but a simple one.

Reply:
On that page the `four-images.csv` is linked but isn't mentioned at all. Do users need to edit it?
I'll add an example csv and also link to the training-scripts idr0079 prep doc for full set-up

Reply:
syrry, my bad. The four-images.csv is used in https://omero-guides.readthedocs.io/en/latest/upload/docs/metadata-ui.html - the page I sent you to has the link to it for historical reasons only